src/components/Header/Header.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 
+import { Link, useHistory } from 'react-router-dom';
+
+
+import profile from '../../assets/svg/user-solid.svg';
+
 import styles from './Header.module.css';
 
 const Header = () => {
+  const history = useHistory()
   return (
     <header className={styles.Header}>
-      <h1 className="Title">Mój Album</h1>
+      <h1 className="Title" style={{ cursor: 'pointer' }} onClick={() => {
```

Comment:
Jak coś stylowanie inline przez style="" powinno być ostatecznością ;) Zwłaszcza jeśli i tak importujesz style (Header.module.css).

Reply:
Oki, poprawię to! Zrobiłam tak na szybko, bo chciałam mieć jakieś przekierowanie na stronę główną z innych komponentów :)

Reply:
Stylowanie inline też już poprawione, tyle że na innym branchu ;) Feature 8